yandex/resource_yandex_compute_instance.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	yandexComputeInstanceDefaultTimeout       = 5 * time.Minute
+	yandexComputeInstanceDefaultTimeout       = 10 * time.Minute
```

Comment:
Its better to introduce a separate timeout for create operation and use it.
For both cases, disk and instance.

Reply:
I'm not sure there is a reason to separate this value. This is only timeout value and doesn't depend on other things.